app/Http/Controllers/Api/WaitingRoomsController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Models\WaitingRoom;
+use Illuminate\Http\Request;
+
+class WaitingRoomsController extends ApiController
+{
+    public function calculateSplit(Request $request)
```

Comment:
But it's for internal stuff. Phoenix would never have to hit this endpoint. Nothing we'd add to the API documentation.

I'd rather keep stuff with controllers related to the actual API w/in the Controllers/Api...

The JS script could very well just hit a dedicated route that's part of the waitingroom and lives as a method on the regular WaitingRoom controller like:
```php
Route::post('waitingrooms/{waitingrooms}/split/calculate', 'WaitingRoomsController@splitCalculation')->name('split');
```

Or something like the above.